static/css/restyle.less

```diff
@@ -580,6 +580,15 @@ button.search-button {
   margin-top: 0;
 }
 
+.toplist li a {
+  line-height: 1.4em;
+  padding: .125em 0 .25em;
```

Comment:
Is this much precision necessary? 

I don't know what the rest of this file looks like but I don't think we should worry about stripping leading zeros e.g `0.25em` seems more obvious and less ambiguous compared to `.25em` but maybe that's just me?